### PR TITLE
Skip images together with links

### LINKED ISSUE #611: Bug: `Proper Ellipsis` rule breaks transcluded links
## Describe the Bug

`Proper Ellipsis` rule breaks transcluded links

## How to Reproduce

``` markdown
![title](a...b.md)

![](c...d.png)
```

Apply `Proper Ellipsis` rule

## Expected Behavior

The note is unchanged

## Actual behavior Behavior

The three consecutive dots `...` were replaced with a single Unicode character `…`, breaking the links

``` markdown
![title](a…b.md)

![](c…d.png)
```

### PR DESCRIPTION
**Pull request by mnaoumov**
Fixes #611

### CODE REVIEW

File: src/utils/ignore-types.ts [hunk 1]
```diff
@@ -132,7 +132,7 @@ function replaceRegex(text: string, placeholder: string, regex: RegExp): IgnoreR
  * @return {string[]} The regular markdown links replaced
  */
 function replaceMarkdownLinks(text: string, regularLinkPlaceholder: string): IgnoreResults {
-  const positions: Position[] = getPositions(MDAstTypes.Link, text);
+  const positions: Position[] = getPositions([MDAstTypes.Link, MDAstTypes.Image], text);
```
Comment (pjkaufman) [inline src/utils/ignore-types.ts:135]: Why add `image` to this list as opposed to adding it to the ignore types list for the function?

Comment (mnaoumov) [reply src/utils/ignore-types.ts:135]: @pjkaufman no reason :)

Comment (mnaoumov) [reply src/utils/ignore-types.ts:135]: I will fix it

**Review by pjkaufman — APPROVED** [review]
Looks good to me.

**Pull request merged**